blocks/ads-block/features/ads/_children/AdUnit/index.jsx

```diff
@@ -27,7 +28,12 @@ const AdUnit = (props) => {
 		if (!isAdmin) registerAd();
 	}, [registerAd, isAdmin]);
 
-	return !isAdmin ? <div id={id} className={`arcad ad-${adClass}`} /> : null;
+	return !isAdmin ? (
```

Comment:
todo: remove isAdmin from this logic. this component will only be shown with admin

Reply:
done